doc/misc/example.adoc

```diff
@@ -8,51 +8,51 @@ The example describes a sensor model FMU with one input, one output, and no addi
 <?xml version="1.0" encoding="UTF-8"?>
 <fmiModelDescription
   fmiVersion="2.0"
-  modelName="OSI Sensor Model Packaging Demo FMU"
+  modelName="OSI Simulation Model Packaging Demo FMU"
   guid="aabc2174e20f08597cfae6947c96bf86"
   variableNamingConvention="structured">
   <CoSimulation
     modelIdentifier="OSMPDemoFMU"
     canNotUseMemoryManagementFunctions="true"/>
   <DefaultExperiment startTime="0.0" stepSize="0.020"/>
   <VendorAnnotations>
-    <Tool name="net.pmsf.osmp" xmlns:osmp="http://xsd.pmsf.net/OSISensorModelPackaging"><osmp:osmp version="1.2.0" osi-version="3.4.0"/></Tool>
+    <Tool name="net.pmsf.osmp" xmlns:osmp="http://xsd.pmsf.net/OSISimulationModelPackaging"><osmp:osmp version="1.2.0" osi-version="3.4.0"/></Tool>
```

Comment:
@pmai please have a look if the changes in the VendorAnnotations for the example fmus are fine for you (just a result of replace all...).

Reply:
None of this can be done in a minor release, since it completely changes the syntax of the standard (different namespace). If we are going to do this, the namespace should be changed to an ASAM-owned domain along the way.

Reply:
But it is not wrong :)
@engelben would we have a suitable domain at ASAM reserved for this?